test/passwords-test.js

```diff
@@ -0,0 +1,26 @@
+const {assert} = require('./common');
+const Passwords = require('../passwords');
+
+describe('Passwords', () => {
+  it('should be able to generate and compare hashes', async () => {
+    const pass = 'apple';
+    const passFake = 'orange';
+
+    const passHash = await Passwords.hash(pass);
+    const passHashSync = Passwords.hashSync(pass);
+    const passFakeHashSync = Passwords.hashSync(passFake);
+
+    assert(Passwords.compareSync(pass, passHash),
+      'hash should compare correctly to original password');
+    const compareAsync = await Passwords.compare(pass, passHash);
+    assert(compareAsync,
```

Comment:
^ sorry can you switch this to expect/jest style assertions ? (I landed that PR)